### ISSUE TITLE
26.1

### ISSUE THREAD

**Issue by eedahl** [zklhp/emacs-w64#7]
Hi! A new version is out? Any chance you would do an update? :)

**Comment by jdtw** [comment]
FWIW, I got a windows build from here: http://alpha.gnu.org/gnu/emacs/pretest/windows/

Edit: Official version is up on the main GNU ftp: http://ftp.gnu.org/gnu/emacs/windows/emacs-26.1-x86_64.zip

**Comment by zklhp** [comment]
Yes, the official version is good. There is of little need to update 😹

**Comment by eedahl** [comment]
Very well :) maybe update the readme to say that haha

**Comment by kshenoy** [comment]
> Edit: Official version is up on the main GNU ftp: http://ftp.gnu.org/gnu/emacs/windows/emacs-26.1-x86_64.zip

The correct link is http://ftp.gnu.org/gnu/emacs/windows/emacs-26/emacs-26.1-x86_64.zip
